ts/Core/Chart/StockChart.ts

```diff
@@ -692,6 +692,7 @@ addEvent(Axis, 'afterDrawCrosshair', function (
             crossLabel
                 .attr({
                     fill: options.backgroundColor ||
+                        point && point.color || // #14888
```

Comment:
Q: Is this expected that only `crossLabel` has the same color as the last point? How about the line?

Check out:
- `series.colorUp`: https://jsfiddle.net/BlackLabel/ba1m7edk/3/
- `point.color`: https://jsfiddle.net/BlackLabel/ba1m7edk/2/ (less popular case, than the top one)

Reply:
Let's stick for now with the code which set color for the line and label from the series.
Changed in the latest commit.